tests/integration/CapabilityTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Automattic\LegacyRedirector\Tests;
+
+use Automattic\LegacyRedirector\Capability;
+
+/**
+ * CapabilityTest class
+ */
+class CapabilityTest extends TestCase {
+
+	/**
+	 * We remove the Capability::MANAGE_REDIRECTS_CAPABILITY  before testing
+	 *
+	 * @beforeClass
+	 */
+	public static function beforeClassSetup() {
+		$capability = new Capability();
+		$capability->unregister();
+	}
+
+	public function test_admin_capabilities_before_register() {
+		$user_id = $this->factory->user->create( array( 'role' => 'administrator' ) );
+		$user    = wp_set_current_user( $user_id );
+
+		$this->assertFalse( boolval( $user->has_cap( Capability::MANAGE_REDIRECTS_CAPABILITY ) ) );
```

Comment:
`->has_cap()` only returns a boolean `true` or `false`, so `boolval()` is redundant here.

Reply:
This is removed now.